### PR TITLE
Try to ensure perf uses as little CPU as possible.

### PR DESCRIPTION
**Pull request by vext01**
Set perf sample rate to minimum value, this minimising time spent profiling.

### CODE REVIEW

File: krun/platform.py [hunk 1]
```diff
@@ -238,6 +239,27 @@ def check_preliminaries(self):
         self._check_cpu_isolated()
         self._check_cpu_governor()
         self._check_cpu_scaler()
+        self._check_perf_samplerate()
+
+
+    def _check_perf_samplerate(self):
+        """Attempt to minimise timp spent by the Linux perf kernel profiler.
```
Comment (ltratt) [inline krun/platform.py]: Spelling (timp).

File: krun/platform.py [hunk 2]
```diff
@@ -238,6 +239,27 @@ def check_preliminaries(self):
...
+        You can't disable this, so the best be can do is set the sample
```
Comment (ltratt) [inline krun/platform.py]: be -> we

**Comment by vext01** [comment]
Fixed. OK?

**Comment by ltratt** [comment]
Yep!

**Pull request merged**